app/frontend/src/uploadDocuments.js

```diff
@@ -1,8 +1,22 @@
-// This triggers a file input when the button is clicked
-
 document.addEventListener('DOMContentLoaded', function() {
-  document.getElementById('file-upload').addEventListener("click", function(e) {
+  inputFileUpload = document.getElementById('documents-form-documents-field')
```

Comment:
You don't need a ternary operator here and you should declare the variable (`const` is better as it's scoped and cannot be reassigned, but `var` is fine). 

```
const inputFileUpload = document.getElementById('documents-form-documents-field') || document.getElementById('documents-form-documents-field-error');
```

Reply:
Er, `const` isn't supported on IE10 and below, which rules out three users in the last three months...

Reply:
`const` gets transpiled into `var` so it's fine.